examples/refrigerator-app/silabs/src/EventHandlerLibShell.cpp

```diff
@@ -113,7 +114,7 @@ CHIP_ERROR RefrigeratorDoorEventHandler(int argc, char ** argv)
     int value = std::stoi(argv[0]); // Safe to use now, as we validated the input earlier
 
     RefrigeratorAlarmEventData * data = Platform::New<RefrigeratorAlarmEventData>();
-    data->eventId                     = Events::Notify::Id;
+    data->eventId                     = RefrigeratorAlarm::Events::Notify::Id;
```

Comment:
Is `RefrigeratorAlarm` necesary if we have `using namespace Clusters::RefrigeratorAlarm;`?
I'm not sure we need both and if we do why is it the case?